modules/javafx.base/src/main/java/javafx/beans/binding/ObjectBinding.java

```diff
@@ -63,6 +63,7 @@
 
     private T value;
     private boolean valid = false;
+    private boolean isObserved;
```

Comment:
Minor: "observed" might be more consistent with the field naming convention in this file.